src/box/txn.c

```diff
@@ -755,13 +755,25 @@ txn_complete_fail(struct txn *txn)
 	rmean_collect(rmean_box, IPROTO_ROLLBACK, 1);
 }
 
+/**
+ * Commit and finalize the transaction.
+ *
+ * This function will run any on_commit triggers associated with @a txn and only
+ * them, so if any substatements of this transaction have on_commit triggers,
```

Comment:
We write descriptions next to function declarations, not definitions - please, move the description there.
Also, `substatements` -> `statements`.

Reply:
done